benchmark/parse/filter.benchmark.cpp

```diff
@@ -14,7 +14,8 @@ using namespace mbgl;
 style::Filter parse(const char* expression) {
     rapidjson::GenericDocument<rapidjson::UTF8<>, rapidjson::CrtAllocator> doc;
     doc.Parse<0>(expression);
-    return *style::conversion::convert<style::Filter, JSValue>(doc);
+    style::conversion::Error error;
+    return *style::conversion::convert<style::Filter, JSValue>(doc, error);
```

Comment:
This is used in the inner loop of one of the two benchmarks, so don't we want to avoid any error checking overhead?